README.md

```diff
@@ -5,38 +5,99 @@
 
 This project is a modern implementation of the Ubuntu Desktop installer, using [subiquity](https://github.com/canonical/subiquity) as a backend and [Flutter](https://flutter.dev/) for the UI.
 
-## Checkout & Run
+![Preview Screenshot](.github/docs/images/preview_screenshot.png)
 
-Grab the source:
+## Availability
+
+The new Ubuntu Desktop installer is *planed* to ship with the next Ubuntu Desktop LTS 22.04 Impish Idri.
+
+## Test the new Ubuntu Desktop installer
+
+If you want to test the new installer before 22.04 goes live, you can download a preview canary image below and install it for example in a virtual machine:
+
+[Download](http://cdimage.ubuntu.com/impish/daily-canary/current/)
+
+
+## Building and running manually
+
+To build the installer manually you need to have the [Flutter UI toolkit](https://flutter.dev) installed and the source code in this repository downloaded on your computer.
+
+To install Flutter you have two options: a manual installation by cloning the flutter repo and adding it tou your $PATH or installing the Flutter snap.
```

Comment:
Suggestion:
> To install Flutter, you have two options: from the [Snap Store](https://snapcraft.io/flutter), or [manually](https://flutter.dev/docs/get-started/install/linux).

I'd mention the snap first because we want people to use it. Furthermore, both of those pages have installation instructions that are unnecessary to repeat here? It would also reduce the noise below so we could get straight to the point. ;)

Reply:
Where did the snap disappear? :)

Reply:
I thought you want to forward people to the official google instructions? On those people find instructions for both manual and snap installations? No? :D As you wish